extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmConfigPersistenceUnit.java

```diff
@@ -664,9 +664,38 @@ interface HibernateOrmConfigPersistenceValidation {
 
         /**
          * Enables the Bean Validation integration.
+         *
+         * @deprecated Use {@link #modes()} instead.
          */
+        @Deprecated(since = "3.19", forRemoval = true)
         @WithDefault("true")
         boolean enabled();
+
+        /**
+         * Defines how the Bean Validation integration behaves.
+         */
+        @WithDefault("AUTO")
+        Set<ValidationMode> modes();
```

Comment:
I would go with `mode`. It reads better and I don't think people will ever use two options at the same time, given `auto` does the job.

And even if you use two values, it's still the `mode` you want to use.